models/classes/class.ItemsService.php

```diff
@@ -132,6 +135,39 @@ public function isItemClass(core_kernel_classes_Class $clazz)
         return $clazz->equals($this->getRootClass()) || $clazz->isSubClassOf($this->getRootClass());
     }
 
+    // @todo Recheck name
+    // @todo Create command class
+    public function delete(array $command): void
```

Comment:
We should call this method inside `deleteResource` https://github.com/oat-sa/extension-tao-item/pull/632/files#diff-946097af7291b432b408417519ac98e9eaef8253771b6880c01a7523a3417104R186 and deprecated to avoid code duplication